tests/012-test-container-root-lv-options.sh

```diff
@@ -0,0 +1,74 @@
+source $SRCDIR/libtest.sh
+
+# Test CONTAINER_ROOT_LV_NAME, CONTAINER_ROOT_LV_MOUNT_PATH and CONTAINER_ROOT_LV_SIZE directives.
```

Comment:
We don't seem to be making any checks for CONTAIER_ROOT_LV_SIZE. So I don't think we are testing for that explicitly.